src/main/kotlin/org/sert2521/bunnybots2019/beddumper/BedDumper.kt

```diff
@@ -0,0 +1,31 @@
+package org.sert2521.bunnybots2019.beddumper;
+
+import org.sert2521.bunnybots2019.MotorControllers
+import org.sert2521.sertain.motors.MotorController
+import org.sert2521.sertain.subsystems.Subsystem
+
+class Beddumper : Subsystem("Beddumper") {
+        private val intakeMotor = MotorController(
```

Comment:
```suggestion
        private val conveyorMotor = MotorController(
```

Since this motor controller refers to the motor that runs the conveyor belt that dumps the bed, we want a variable name that reflects that.

Reply:
There will be a few lines in your code that you'll need to change after renaming this. I think you'll be able to find them pretty easily though so I won't highlight them here 😄